### PR TITLE
Hide missing criteria table

### PR DESCRIPTION
**Pull request by lucyxubroad**
_Pull requests into Flux require the following. Submitter and reviewer should :white_check_mark: when done. For items that are not-applicable, note it's not-applicable (**N/A**) and :white_check_mark:._

Addresses JIRA 1172
- NOTE: The JIRA task was originally to change the "None" text depending on whether or not a trial is selected. For this pull request, I added a functionality to hide the "Missing Criteria" table altogether until you click to view missing criteria of a trial, which negates the need for the original JIRA task.
- Added "displayFunction" to Missing Criteria table in SummaryMetadata.jsx, which indicates whether or not the table should be rendered. If the boolean value is false, the renderSubsection function in TabularVisualizer.jsx would return an empty div.
- Changed the PATINA FHIR patient to show a different missing criteria.  

## Submitter: Lucy

**Running the Application**

- [x] Manually tested in Chrome
- [x] Manually tested in IE

**Documentation**

- [x] This pull request describes why these changes were made
- [x] Recognizes any potential shortcomings/bugs in the description 
- Cheat sheet is updated
- Demo script is updated 
- Documentation on Wiki has been updated 
- Additional technical components and libraries are documented and added to wiki as needed

**NoteParser**

- Note parser has been updated if structured phrases change

**Code Quality**

- [x] 4-space indents - convert any tabs to spaces
- [x] Use public class field syntax for binding functions - ex. handleChange = () => { ... };
- [x] Code is commented

**Tests**

- [x] Existing tests passed
- Added Enzyme-UI tests for slim mode 
- Added Enzyme-UI tests for full mode
- Added backend tests for new functionality added
- Added note parser examples to test new functionality

## Reviewer 1: Name

- [ ] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [ ] The tests appropriately test the new code, including edge cases
- [ ] You have tried to break the code

## Reviewer 2: Name

- [ ] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [ ] The tests appropriately test the new code, including edge cases
- [ ] You have tried to break the code

### CODE REVIEW

**Comment by lucyxubroad** [comment]
By the way, I did add a display functionality to SummaryMetadata for the Missing Criteria table. Should I update the wiki at all for this change?

**Comment by nicoleng12** [comment]
Looks good! Backend tests pass and ran through the demo script which also passed. Tested in IE and chrome.

File: src/summary/SummaryMetadata.jsx [hunk 1]
```diff
@@ -1000,7 +1002,11 @@ export default class SummaryMetadata {
     }
 
     getMissingCriteriaSubsectionName = () => {
-        return `Missing ${this.trialDisplayMissingCriteria} Criteria`;
+        return (this.trialDisplayMissingCriteria !== "") ? (`Missing ${this.trialDisplayMissingCriteria} Criteria`) : "Missing Criteria";
+    }
+
+    getMissingCriteriaDisplay = () => {
+        return (this.trialDisplayMissingCriteria !== "") ? true : false;
```
Comment (danlee1025) [inline src/summary/SummaryMetadata.jsx]: I think this line can be changed to ```return this.trialDisplayMissingCriteria !== "";```

Seems kind of redundant to use a ternary operator here.

**Comment by lucyxubroad** [comment]
most recent push addresses DL1
